docs/source/guides/using_compose_with_featuretools.ipynb

```diff
@@ -938,7 +938,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "y = feature_matrix.pop(labels.label_name)\n",
+    "y = feature_matrix.pop(labels.target_columns[0])\n",
```

Comment:
I think it might be a little more clear to the reader if you pop using the actual label name here instead of `target_columns[0]`, which could be confusing.